ivadomed/postprocessing.py

```diff
@@ -161,3 +162,48 @@ def mask_predictions(predictions, mask_binary):
     assert predictions.shape == mask_binary.shape
     assert np.array_equal(mask_binary, mask_binary.astype(bool))
     return predictions * mask_binary
+
+
+def coordinate_from_heatmap(nifti_image, thresh=0.3):
+    """
+    Retrieve coordinates of local maxima in a soft segmentation.
+    Args:
+        nifti_image (nibabel object): nifti image of the soft output.
```

Comment:
the concept of input/output is relative to where we are (in this function, it is an input, not an output, so it could be confusing)
```suggestion
        nifti_image (nibabel object): nifti image of the soft segmentation.
```